docs/ossm/injection/README.md

```diff
@@ -1,27 +1,39 @@
 ## Installing the Sidecar
 ### Injection
-In order to take advantage of all of Istio’s features, pods in the mesh must be running an Istio sidecar proxy.
+To include workloads as part of the service mesh and to begin using Istio's many features, pods must be injected with a sidecar proxy that will be managed by an Istio control plane.
 
-Automatic Istio sidecar injection can be enabled via labels on a namespace level or directly for a given pod. Automatic injection injects the proxy configuration at pod creation time using an [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/).
+Sidecar injection can be enabled via labels at the namespace or pod level that also serve to identify the control plane managing the sidecar proxy(ies). By adding a valid pod injection label (see below) on a namespace, any new pods that are created in that namespace will automatically have a sidecar added to them.
 
-> **_NOTE:_** Upstream Istio offers also Manual injection using the ` istioctl kube-inject` which is not supported by OpenShift Service Mesh.
+The proxy configuration is injected at pod creation time using an [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/). As sidecar injection occurs at the pod-level, you won’t see any change to `Deployment` resources. Instead, you’ll want to check individual pods (via `oc describe`) to see the injected Istio proxy container.
 
-After adding a correct label (see below) on a namespace, any new pods that are created in that namespace will automatically have a sidecar added to them.
-
-Automatic injection occurs at the pod-level. You won’t see any change to the deployment itself. Instead, you’ll want to check individual pods (via `oc describe`) to see the injected proxy.
-
-### Controlling the injection policy
-When a `default` control plane revision is installed or a default tag exists, it's possible to use following labels on a namespace or a pod:
+### Enabling sidecar injection
+When the service mesh's `Istio` resource is created with the name `default`, it's possible to use following labels on a namespace or a pod:
```

Comment:
This is not true in all cases. Istio resource can be called `default` but with `RevisionBased` upgrade strategy the `istio-injection` label will not work. That's why I used "`default` control plane revision" as the IstioRevision must be named `default`. This will no longer be true when the support for tags is added, then it will be possible to use the default tag. I will add more examples so it's clear.

Reply:
Moved to https://github.com/openshift-service-mesh/sail-operator/pull/152